host/src/lib.rs

```diff
@@ -88,6 +103,10 @@ impl Opts {
         "0.0.0.0:8080".to_string()
     }
 
+    fn default_sp1_worker_address() -> String {
+        "0.0.0.0:8081".to_string()
```

Comment:
```suggestion
        // Placeholder address
        "0.0.0.0:8081".to_string()
```

AFAIK in some cases 0.0.0.0 is/was use for broadcasting so I think it should be clarified it's a placeholder until we have a valid address.